.babelrc

```diff
@@ -1,4 +1,10 @@
 {
-  "presets": ["es2015", "react"],
-  "plugins": ["transform-class-properties", "transform-object-rest-spread"]
+  "presets": [
+    "env",
```

Comment:
Note that in this PR I also use some transforms directly as a hack while we're still working out native support for them: https://github.com/facebook/prepack/pull/397/files#r114634310